docker/Dockerfile

```diff
@@ -5,34 +5,55 @@ LABEL MAINTAINER IOHK
 LABEL description="Jormungandr latest"
 
 ARG PREFIX=/app
-ARG REST_PORT=8448
-
 ENV ENV_PREFIX=${PREFIX}
+ARG REST_PORT=8448
+ARG BUILD=false
+ENV ENV_BUILD=${BUILD}
+ARG VER=v0.2.3
+ENV ENV_VER=${VER}
 
 # prepare the environment
 RUN apt-get update && \
-    apt-get install -y build-essential pkg-config git libssl-dev curl && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
-# install the node and jcli from source
-RUN mkdir -p ${ENV_PREFIX} && \
+    apt-get install -y git curl && \
+    mkdir -p ${ENV_PREFIX} && \
     mkdir -p ${ENV_PREFIX}/src && \
     mkdir -p ${ENV_PREFIX}/bin && \
+    cd ${ENV_PREFIX} && \
+    git clone --recurse-submodules https://github.com/input-output-hk/jormungandr src && \
```

Comment:
this is just cloned to get the script files in the case of build=false .. that seems a bit overkill seems curl should grab them from the repository in raw mode instead